src/app/publications/page.tsx

```diff
@@ -13,22 +14,50 @@ import Sidebar from '@/components/SideBar'
 import Divider from '@/components/Divider'
 import { ScreenSize, linearlyScaleSize } from '@/app/theme'
 
+const CenteredContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
```

Comment:
Why do we need this component? It looks like, without this div, the rendered scene looks the same.

Reply:
That was my code. I think I was trying to center the page in widescreen and this ended up being redundant.